cmd/assetsvc/handler.go

```diff
@@ -71,17 +71,19 @@ type count struct {
 	Count int
 }
 
-// getPageNumberAndSize extracts the page number and size of a request. Default (1, 0) if not set
-func getPageNumberAndSize(req *http.Request) (int, int) {
-	page := req.FormValue("page")
-	size := req.FormValue("size")
-	pageInt, err := strconv.ParseUint(page, 10, 64)
+// getPageAndSizeParams extracts the page number and the page size of a request. Default (page,size) = (1, 0) if not set
+func getPageAndSizeParams(req *http.Request) (int, int) {
+	pageNumber := req.FormValue("page")
+	pageSize := req.FormValue("size")
+
+	pageNumberInt, err := strconv.ParseUint(pageNumber, 10, 64)
 	if err != nil {
```

Comment:
I would handle the case in which the page is `0` and set it to `1` in that case.